staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -219,6 +220,12 @@ func (t *tracker) List(gvr schema.GroupVersionResource, gvk schema.GroupVersionK
 		return nil, err
 	}
 
+	// If there are no items in the store we return list resourceVersion "1" as "0" and "" are reserved values.
+	err = meta.NewAccessor().SetResourceVersion(list, "1")
```

Comment:
Seems like this is covered down below

Reply:
yeah, it can bail out for empty list is store is empty

Reply:
here: https://github.com/kubernetes/kubernetes/pull/74898/files#diff-1fedd5b881e8fbda5aaefa9bd74b359bR238

Reply:
move to the single callsite?

Reply:
I'd prefer not to. It is not the property of the call site to set this initial value. The list object should have a valid RV throughout the whole body. It's also more bug prone if you set it directly in the branch:
```golang
	objs, ok := t.objects[gvr]
	if !ok {
               	// If there are no items in the store we return list resourceVersion "1" as "0" and "" are reserved values.
		err = meta.NewAccessor().SetResourceVersion(list, "1")
		if err != nil {
			return nil, err
		}
		return list, nil
	}
```
and someone  would prefix it e.g.
```golang
	if  t.objects != nil {
		return list, nil
	}
```
he could either forget to set the RV there or would be duplicating the logic.